package.json

```diff
@@ -17,16 +17,22 @@
     "license": "ISC",
     "homepage": "",
     "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "^5.5.0",
+        "@typescript-eslint/parser": "^5.5.0",
         "clean-webpack-plugin": "^3.0.0",
         "copy-webpack-plugin": "^7.0.0",
         "css-loader": "^5.1.0",
-        "eslint": "^7.27.0",
+        "eslint": "^7.32.0",
```

Comment:
I tried to run npm run lint, but it fall with errors.
Could you please check if it runs successfully on your local machine?

Reply:
Checked for errors with lint, eliminated all lint and prettier warnings and errors